packages/@react-aria/calendar/test/useCalendar.test.js

```diff
@@ -227,4 +239,28 @@ describe('useCalendar', () => {
       await testPagination(defaultValue, rangeBefore, rangeAfter, rel, count, visibleDuration, pageBehavior);
     });
   });
+
+  describe('firstDayOfWeek', () => {
+    it.each`
+    Name | defaultValue | firstDayOfWeek | expectedFirstDay | locale
+    ${'default'} | ${new CalendarDate(2024, 1, 1)} | ${undefined} | ${'Sunday, December 31, 2023'} | ${'en-US'}
+    ${'Sunday'} | ${new CalendarDate(2024, 1, 1)} | ${'sun'} | ${'Sunday, December 31, 2023'} | ${'en-US'}
+    ${'Monday'} | ${new CalendarDate(2024, 1, 1)} | ${'mon'} | ${'Monday, December 25, 2023'} | ${'en-US'}
```

Comment:
This doesn't seem correct, it should be Jan 1. I'm seeing the correct behavior in storybook though. Investigating.

Reply:
maybe need a newer Node version? it passed for me locally in `v18.20.3`